### ISSUE TITLE
vendoring is incomplete

### ISSUE THREAD

**Issue by powerman** [tcnksm/ghr#65]
Looks like some packages are missing from vendor/ and current versions of these packages can't satisfy dependencies:
```
+ go install -v all
src/github.com/tcnksm/ghr/vendor/github.com/golang/protobuf/descriptor/descriptor.go:46:2: cannot find package "google.golang.org/genproto/protobuf" in any of:
	/go/src/github.com/tcnksm/ghr/vendor/google.golang.org/genproto/protobuf (vendor tree)
	/usr/local/go/src/google.golang.org/genproto/protobuf (from $GOROOT)
	/go/src/google.golang.org/genproto/protobuf (from $GOPATH)
src/github.com/tcnksm/ghr/vendor/golang.org/x/oauth2/google/default.go:16:2: cannot find package "cloud.google.com/go/compute/metadata" in any of:
	/go/src/github.com/tcnksm/ghr/vendor/cloud.google.com/go/compute/metadata (vendor tree)
	/usr/local/go/src/cloud.google.com/go/compute/metadata (from $GOROOT)
	/go/src/cloud.google.com/go/compute/metadata (from $GOPATH)
```

**Comment by timfallmk** [comment]
@powerman See if this solves your problem.

**Comment by powerman** [comment]
@timfallmk If you'll fix build on CI and @tcnksm is willing to accept your PR in general - I can then try to test your PR too.

**Comment by timfallmk** [comment]
@powerman Interesting, I don't get that error locally. I'll take a look.

Ultimately, the CI won't pass because of missing a GitHub API token, but that's true of all PR's I see here.

**Comment by timfallmk** [comment]
Ah, I see, just a typo I didn't commit. All fixed now.

**Comment by Songmu** [comment]
Resolved at #81. Thanks for reporting!
